kobo/apps/trackers/utils.py

```diff
@@ -49,14 +50,15 @@ def update_nlp_counter(
         counter_id = counter.pk
 
     # Update the total counters by the usage amount to keep them current
+    deduct = settings.STRIPE_ENABLED
     kwargs = {}
     if service.endswith('asr_seconds'):
         kwargs['total_asr_seconds'] = F('total_asr_seconds') + amount
-        if asset_id is not None and organization is not None:
+        if deduct and asset_id is not None and organization is not None:
```

Comment:
Since we're touching these lines, would you mind reevaluating these boolean checks? I'm not entirely sure why we are checking asset_id here. And is it possible for organization to be None at this point?